docs/reference/module-options.md

```diff
@@ -0,0 +1,148 @@
+# Module configuration options
+
+Most Apostrophe modules, both core and project-level, can be configured with settings that influence functionality without having to write custom Javascript. The sections below describe the options available in all modules as well as those specific to certain module types.
+
+All the settings described here are placed in a module's `options` configuration object. This often added in the respective `index.js` file, but can also be added where the module is instantiated in the `app.js` file.
+
+`index.js` example:
+```javascript
+// In `modules/article/index.js`
```

Comment:
I think we're sometimes inconsistent here. Sometimes we just put the location at the very top of the code block like

`// modules/article/index.js`

Here, we have "In" with quotes.

I prefer the former, but thing consistency is the most important